invenio_rdm_records/services/schemas/record.py

```diff
@@ -28,7 +28,6 @@
 from marshmallow_utils.permissions import FieldPermissionsMixin
 
 from .access import AccessSchema
-from .endorsements import EndorsementItemSchema
 from .files import FilesSchema
 from .metadata import MetadataSchema
 from .parent import RDMParentSchema
```

Comment:
better to following pattern current pattern, endorsement better have his own file